app/assets/javascripts/application.js

```diff
@@ -218,12 +218,12 @@ $(document).on('change', '#panel-genes-search input, #panel-genes-search select'
 
 // split a string on spaces, used for extractLast()
```

Comment:
update the comment to reflect that this also does commas now

Reply:
d9d00de